recipes/gettext/all/conandata.yml

```diff
@@ -24,6 +24,9 @@ patches:
       patch_description: "apply the __atribute_noreturn__ to print_and_abort"
       patch_source: "https://github.com/coreutils/gnulib/commit/0cc39712803ade7b2d4b89c36b143dad72404063"
       patch_type: "conan"
+    - patch_file: "patches/0.21-include-stdlib_h.patch"
+      patch_description: "include stdlib.h for newer gcc"
+      patch_type: "portability"
```

Comment:
If this is solved in 0.22, it would fall under the backport category. We're more likely to prefer this option if you can find the commit that fixed this for that verison and link it in the `patch_source` field. Wdyt @toge?